2-environments/envs/production/outputs.tf

```diff
@@ -38,3 +38,8 @@ output "env_secrets_project_id" {
   description = "Project for environment related secrets."
   value       = module.env.env_secrets_project_id
 }
+
+output "assured_workload_folder_display_name" {
+  description = "Assured Workload folder display name."
+  value       = local.enable_assured_workload ? local.assured_workload_display_name : ""
```

Comment:
`id` maybe more useful for end users

Reply:
Do we need an output at all?

Reply:
@morgante I was thinking of usage in tests (using [gcloud assured workloads describe](https://cloud.google.com/sdk/gcloud/reference/assured/workloads/describe)) as the resource seems to have a generated UUID as name.

Reply:
Ah that makes sense. I agree we should return ID then.

Reply:
the ID of the workload added an an output + the folder id that contains the workload